### ISSUE TITLE
The latest version docs have broken intra-doc links

### ISSUE THREAD

**Issue by hunterwerlla** [aws/aws-cdk#3403]
* **I'm submitting a ...**
  - [ ] :beetle: bug report
  - [ ] :rocket: feature request
  - [ ] :books: construct library gap
  - [ ] :phone: security issue or vulnerability => Please see [policy](https://github.com/aws/aws-cdk/security/policy)
  - [ ] :question: support request => Please see note at the top of this template.
  - [ ] Documentation related issue

* **Other information** (
Some doc pages such as [this one](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-codebuild.LinuxBuildImage.html) have broken intra-doc links. 

For example: "The default {@link ComputeType} to use with this image, if one was not specified in {@link BuildEnvironment#computeType} explicitly."

**Comment by rix0rrr** [comment]
Correct, this is not  supported right now. Thanks for  reporting, we'll put it on the list.
